src/txn/txn_rollback_to_stable.c

```diff
@@ -148,6 +191,31 @@ __rollback_row_add_update(WT_SESSION_IMPL *session, WT_PAGE *page, WT_ROW *rip,
     return (ret);
 }
 
+/*
+ * __rollback_col_ondisk_fixup_key --
+ *     Allocate tombstone and calls function add update to head of insert list.
+ */
+static int
+__rollback_col_ondisk_fixup_key(WT_SESSION_IMPL *session, WT_PAGE *page, WT_REF *ref, WT_COL *cip,
+  wt_timestamp_t rollback_timestamp, bool replace, uint64_t recno)
+{
+    WT_DECL_RET;
+    WT_UPDATE *upd;
+
+    WT_UNUSED(rollback_timestamp);
+    WT_UNUSED(replace);
+    WT_UNUSED(page);
+
+    /* Allocate tombstone to update to remove the unstable value. */
+    WT_ERR(__wt_upd_alloc_tombstone(session, &upd, NULL));
+    WT_STAT_CONN_DATA_INCR(session, txn_rts_keys_removed);
+    WT_ERR(__rollback_col_add_update(session, ref, cip, upd, recno));
+    return (ret);
+
+err:
+    return (ret);
```

Comment:
Need to free the `upd` in case of an error.

Reply:
Fixed.